test/vast_tracker.js

```diff
@@ -538,4 +539,84 @@ describe('VASTTracker', function() {
       });
     });
   });
+
+  describe('#clickthroughs', () => {
+    const fallbackClickThroughURL = 'http://example.com/fallback-clickthrough',
+      clickThroughURL = 'http://example.com/clickthrough';
+
+    describe('#VAST clickThrough with no fallback provided', () => {
+      const eventsSent = [];
+      before(done => {
```

Comment:
No need for `done` to be passed and called, nothing async is going on :) The same for the other tests